enterprise/cmd/frontend/internal/compute/streaming/stream.go

```diff
@@ -109,15 +109,30 @@ LOOP:
 
 	matchesFlush()
 
-	if err = getErr(); err != nil {
+	alert, err := getResults()
+	if err != nil {
 		_ = eventWriter.Event("error", streamhttp.EventError{Message: err.Error()})
 		return
 	}
 
 	if err := ctx.Err(); errors.Is(err, context.DeadlineExceeded) {
 		_ = eventWriter.Event("alert", streamhttp.EventAlert{
-			Title:       "Heads up",
-			Description: "This data is incomplete! We ran this query for 1 minute and we'll need more time to compute all the results. Ask in #compute for more Compute Credits™",
+			Title:       "Incomplete data",
+			Description: "This data is incomplete! We ran this query for 1 minute and we'll need more time to compute all the results.",
```

Comment:
```suggestion
			Description: "This data is incomplete! We ran this query for 1 minute and we'll need more time to compute all the results. Please reach out to XYZ",
```

If there's a chance that this alert will now surface outside the context of Sourcegraph internal users, we should say something appropriate, in terms of support, I suppose!

Reply:
mostly works, added a small rephrase, thanks!